src/current/releases/cloud.md

```diff
@@ -5,7 +5,7 @@ toc: true
 docs_area: releases
 ---
 
-CockroachDB Cloud supports the latest major version of CockroachDB and the version immediately preceding it. All clusters are subject to automatic upgrades to the latest supported minor version. [{{ site.data.products.serverless }}](../cockroachcloud/quickstart.html) clusters are subject to automatic upgrades for both minor and major releases while Serverless is in beta. For more information, see the [{{ site.data.products.db }} Upgrade Policy](../cockroachcloud/upgrade-policy.html).
+{{ site.data.products.db }} supports the latest major version of CockroachDB and the version immediately preceding it. All clusters are subject to automatic upgrades to the latest supported patch version. {{ site.data.products.serverless }} clusters are subject to automatic upgrades for both patch and major releases. For more information, see the [{{ site.data.products.db }} Upgrade Policy](../cockroachcloud/upgrade-policy.html).
```

Comment:
Looking again, I think it might be better to actually delete most of this info and link to the upgrade policy page. Feature releases in cloud are often not tied to CRDB releases, and I don't want to start the page off implying that CRDB upgrades are causing the changes listed below. Pushing changes in a minute, LMK what you think.